lib/tile-importer.js

```diff
@@ -0,0 +1,114 @@
+const mkdirp = require('mkdirp')
+const pump = require('pump')
+const path = require('path')
+const tar = require('tar-fs')
+const mirror = require('mirror-folder')
+const fs = require('fs')
+
+module.exports = TileImporter
+
+function TileImporter (userData, defaults) {
+  if (!(this instanceof TileImporter)) return new TileImporter(userData, defaults)
+  this.editing = false
+  this.defaults = Object.assign({}, TileImporter.defaults, defaults)
+  this.userData = userData
+}
+
+TileImporter.defaults = {
+  name: 'Offline Maps',
+  type: 'tms', // TODO: what is this??
```

Comment:
@noffle @gmaclennan in the future i'd like to allow people to override these basic settings if they want, but for now just trying to pick smart easy defaults. any thoughts for how to be smarter on this is very welcome!

Reply:
@karissa
Just came back from checking my pull request.
What I could find out is that 'tms' is short for Tile Map Service. When assigned in the imagery.json to 'type' it wont make a difference. So I guess this is the default value for these offline tile service definitions.
Evidence:
https://wiki.openstreetmap.org/wiki/TMS

This makes sense to me because all these tiles are later served by a local ecstatic server on localhost:5005

Reply:
Nice thanks